dist/comment_style.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import argparse, os, re, subprocess, sys
+
+def print_msg(file_name, line_num, line, multiline):
```

Comment:
nit: Can we rename to something like `report_illegal_comment`?

Reply:
Yep - done in bf9d525.